src/Tests/HelixTasks/SDKCustomCreateXUnitWorkItemsWithTestExclusion.cs

```diff
@@ -141,11 +142,20 @@ private async Task<List<ITaskItem>> PrepareWorkItem(ITaskItem xunitProject)
             var partitionedWorkItem = new List<ITaskItem>();
             foreach (var assemblyPartitionInfo in assemblyPartitionInfos)
             {
-                string command = $"{driver} exec {assemblyName} {testExecutionDirectory} {msbuildAdditionalSdkResolverFolder} {(XUnitArguments != null ? " " + XUnitArguments : "")} -xml testResults.xml {assemblyPartitionInfo.ClassListArgumentString} {arguments}";
+                var identity = assemblyPartitionInfo.DisplayName + testIdentityDifferentiator;
+                var testResultsDir = IsPosixShell ? @"$HELIX_WORKITEM_UPLOAD_ROOT/" : @"%HELIX_WORKITEM_UPLOAD_ROOT%\";
+
+                string command;
                 if (netFramework)
                 {
                     var testFilter = String.IsNullOrEmpty(assemblyPartitionInfo.ClassListArgumentString) ? "" : $"--filter \"{assemblyPartitionInfo.ClassListArgumentString}\"";
-                    command = $"{driver} test {assemblyName} {testExecutionDirectory} {msbuildAdditionalSdkResolverFolder} {(XUnitArguments != null ? " " + XUnitArguments : "")} --results-directory .\\ --logger trx {testFilter}";
+                    command = $"{driver} test {assemblyName} {testExecutionDirectory} {msbuildAdditionalSdkResolverFolder} {(XUnitArguments != null ? " " + XUnitArguments : "")} --results-directory {testResultsDir} --logger trx {testFilter}";
+                }
+                else
+                {
+                    command  =
+                        $"{driver} exec {assemblyName} {testExecutionDirectory} {msbuildAdditionalSdkResolverFolder} {(XUnitArguments != null ? " " + XUnitArguments : "")} " +
+                        $"-html \"{testResultsDir}{identity}.html\" {assemblyPartitionInfo.ClassListArgumentString} {arguments}";
```

Comment:
It looks like this is no longer outputting the results in xml or trx format.  I thought one of these formats was how the test results show up in the Azure DevOps test results.  Do they still show up correctly with these changes?

Reply:
I have never seen failures show up in AZDO

Reply:
Let me try.

Reply:
This is what I mean.  You haven't seen results like this before?

![image](https://github.com/dotnet/sdk/assets/145043/05d0134e-ad1b-4b17-8f8a-cde21b534262)

Reply:
I see, yes. I think I haven't seen them for dotnet-watch failures because when Helix kills the process there is no output file.
I wonder how does that .xml/trx file get to AzDO though.

Reply:
Reverted that part.